src/Tempest/Http/src/Status.php

```diff
@@ -131,7 +131,7 @@ public function description(): string
             415 => 'Unsupported Media Type',
             416 => 'Range Not Satisfiable',
             417 => 'Expectation Failed',
-            418 => 'I Am A Teapot',
+            418 => "I'm a Teapot",
```

Comment:
```suggestion
            418 => "I'm a teapot",
```

Please be a little serious. The specs say "I'm a teapot", not "I'm a Teapot".